### ISSUE TITLE
Java toolchain failing in Travis.

### ISSUE THREAD

**Issue by shurcooL** [sourcegraph/srclib#152]
Commit 163d4634c8b7470025a1cd1573244d250dd68c27 (/cc @beyang) that added Java toolchain to standard toolchains list is causing Travis build to fail, because the Java toolchain has a problem in its installation. Snippet:

```
downloaded9 KB/478 KB downloaded63 KB/478 KB downloaded7 KB/478 KB downloaded71 KB/478 KB downloaded5 KB/478 KB downloaded8 KB/478 KB downloaded> Configuring > 1/1 projectsaven.org/maven2/org/slf4j/slf4j-api/1.7.2/slf4j-api-1.7.2.jar
> Configuring > 1/1 projects > 4 KB/25 KB downloaded7 KB/25 KB downloaded11 KB/25 KB downloaded5 KB/25 KB downloaded9 KB/25 KB downloaded23 KB/25 KB downloaded5 KB/25 KB downloaded> Configuring > 1/1 projects> Building 0% > :compileJava > Resolving dependencies ':compile'pileJava 
> Building 0%FAILED
> Building 0%> Building 0%25%> Building 25%FAILURE: 
> Building 25%Build failed with an exception.
> Building 25%> Building 25%> Building 25%> Building 25%pileJava'.
> Building 25%> 
> Building 25%> Building 25%> Building 25%> Building 25%> Building 25%--stacktrace
> Building 25%> Building 25%--info
> Building 25%> Building 25%--debug
> Building 25%ore log output.
> Building 25%> Building 25%BUILD FAILED
> Building 25%> Building 25%> Building 25%e: 1 mins 28.206 secs
> Building 25%make[1]: *** [build/libs/srclib-java-0.0.1-SNAPSHOT.jar] Error 1
failed to install/upgrade Java (sourcegraph.com/sourcegraph/srclib-java) toolchain: skipped : 
```

Full build log is at https://travis-ci.org/sourcegraph/srclib/builds/63888635.

**Comment by shurcooL** [comment]
I am guessing this is related to https://github.com/sourcegraph/srclib-java Travis test failing also.

**Comment by shurcooL** [comment]
This was resolved in c7755b4327d772849b3a14f45b767bc91f979418.
